backend/src/services/githubIntegrationService.ts

```
@@ -73,4 +79,105 @@ export default class GithubIntegrationService {
       url: repo.html_url,
     }))
   }
+
+  public static async findOrgDetail(org: string) {
+    const auth = await getGithubInstallationToken()
+    const { data } = await request('GET /orgs/{org}', {
+      org,
+      headers: {
+        authorization: `bearer ${auth}`,
+      },
+    })
+
+    return {
+      description: data.description || null,
+      github: data.html_url,
+      logoUrl: data.avatar_url,
+      name: data.login,
+      twitter: data.twitter_username || null,
+      website: data.blog || null,
+    }
```

NIT: Could data be null/empty here?

---

if the organization is not found or something the api should throw an error, so data should never be null. By the way i could add a try catch for robustness